src/Telegram/Bot/API/Methods.hs

```diff
@@ -213,3 +213,81 @@ toSendDocument ch df = SendDocumentRequest
+getFile = client (Proxy @GetFile)
+
+-- ** 'sendPhoto'
+type SendPhotoContent
+  = "sendPhoto"
+  :> MultipartForm Tmp SendPhotoRequest
+  :> Post '[JSON] (Response Message)
+
+type SendPhotoLink
+  = "sendPhoto"
+  :> ReqBody '[JSON] SendPhotoRequest
+  :> Post '[JSON] (Response Message)
+
+data PhotoFile
+  = PhotoFileId Int
+  | PhotoUrl Text
+  | PhotoFile FilePath ContentType
+
+instance ToJSON PhotoFile where
+  toJSON (PhotoFileId i) = toJSON (show i)
+  toJSON (PhotoUrl t) = toJSON t
+  toJSON (PhotoFile f _) = toJSON ("attach://" <> T.pack (takeFileName f))
+
+-- | Request parameters for 'sendPhoto'
+data SendPhotoRequest = SendPhotoRequest
+  { sendPhotoChatId :: SomeChatId -- ^ Unique identifier for the target chat or username of the target channel (in the format @\@channelusername@).
+  , sendPhotoPhoto :: PhotoFile -- ^ Pass a file_id as String to send a file that exists on the Telegram servers (recommended), pass an HTTP URL as a String for Telegram to get a file from the Internet, or upload a new one using multipart/form-data
+  , sendPhotoThumb :: Maybe FilePath -- ^ Thumbnail of the file sent; can be ignored if thumbnail generation for the file is supported server-side. The thumbnail should be in JPEG format and less than 200 kB in size. A thumbnail's width and height should not exceed 320. Ignored if the file is not uploaded using multipart/form-data. Thumbnails can't be reused and can be only uploaded as a new file, so you can pass “attach://<file_attach_name>” if the thumbnail was uploaded using multipart/form-data under <file_attach_name>
+  , sendPhotoCaption :: Maybe Text -- ^ Photo caption (may also be used when resending Photos by file_id), 0-1024 characters after entities parsing
+  , sendPhotoParseMode :: Maybe ParseMode -- ^ Mode for parsing entities in the Photo caption.
+  , sendPhotoDisableNotification :: Maybe Bool -- ^ Sends the message silently. Users will receive a notification with no sound.
+  , sendPhotoReplyToMessageId :: Maybe MessageId
+  , sendPhotoReplyMarkup :: Maybe SomeReplyMarkup -- ^ Additional interface options. A JSON-serialized object for an inline keyboard, custom reply keyboard, instructions to remove reply keyboard or to force a reply from the user.
+  }
+  deriving Generic
+
+instance ToMultipart Tmp SendPhotoRequest where
+  toMultipart SendPhotoRequest{..} = MultipartData fields files where
+    fields =
+      [ Input "photo" $ T.pack $ "attach://file"
+      , Input "chat_id" $ case sendPhotoChatId of
+          SomeChatId (ChatId chat_id) -> T.pack $ show chat_id
+          SomeChatUsername txt -> txt
+      ] <>
+      (   (maybe id (\_ -> ((Input "thumb" "attach://thumb"):)) sendPhotoThumb)
+        $ (maybe id (\t -> ((Input "caption" t):)) sendPhotoCaption)
+        $ (maybe id (\t -> ((Input "parse_mode" (TL.toStrict $ encodeToLazyText t)):)) sendPhotoParseMode)
+        $ (maybe id (\t -> ((Input "disable_notifications" (bool "false" "true" t)):)) sendPhotoDisableNotification)
```

Comment:
There was a type in initial sendDocument code, and this code is also affected.
It should be `disable_notification` here

Reply:
@qnikst fixed and also corrected in `sendDocument` :+1: